ansible/group_vars/all/defaults.yaml

```diff
@@ -5,17 +5,17 @@ python_path: ""
 # If it's not there, the kubernetes_full_version will have "None" for a version number.
 #
 # IMPORTANT When you update kubernetes_version, also update crictl_version.
-kubernetes_version: "1.29.6"
+kubernetes_version: "1.30.5"
 kubernetes_major_minor: "{{ (kubernetes_version.split('.') | map('trim'))[:2] | join('.') }}"
 kubernetes_semver: "v{{ kubernetes_version }}"
 
-kubernetes_cni_version: "1.3.0"
+kubernetes_cni_version: "1.4.0"
 
 # The crictl CLI is released as part of the http://sigs.k8s.io/cri-tools project.
 # The project release closely follows the Kubernetes release cycle, and uses a
 # nearly identical version scheme.
 # IMPORTANT When you update crictl_version, also update crictl_sha256.
-crictl_version: "{{ kubernetes_major_minor }}.0"
+crictl_version: "{{ kubernetes_major_minor }}.1"
```

Comment:
On line 24 the SHA needs to change too 

https://github.com/mesosphere/konvoy-image-builder/actions/runs/11039503484/job/30666394971?pr=1188#step:9:1368

Reply:
updated !!
https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.30.1

Reply:
Thanks!